### ISSUE TITLE
Settings: Keyboard focus is not visible after tabbing away from toggle control

### ISSUE THREAD

**Issue by chrisgurney** [chrisgurney/obsidian-note-toolbar#501]
_[This is a bug](https://discord.com/channels/686053708261228577/716028884885307432/1454335099545059389) with Obsidian. This issue is to record the workaround fix for Note Toolbar._ 

With respect to settings keyboard accessibility in Obsidian 1.11.x, when you tab between settings elements that use a toggle, there's an invisible element that's capturing focus.

## Steps to reproduce
1. In Obsidian's settings, click on the label of the **General → Automatic updates** setting.
2. Press `tab` so that the toggle becomes active.
3.  Press `tab` again.

## Result
Nothing appears active.

## Expected result
The next setting control (e.g., the _Receive early access versions_ toggle in this case) should be active.

## Analysis
If you watch `document.activeElement` in the console, it looks like it's because the inner input (also with a `tabindex="0"`) is getting the focus, even though it's not visible:
1. `tab` → (correct) `<label class="checkbox-container is-enabled" tabindex="0"><input type="checkbox" tabindex="0"></label>`
2. `tab` → (incorrect) `<input type="checkbox" tabindex="0">`
3. `tab` → (correct) `<label class="checkbox-container is-enabled" tabindex="0"><input type="checkbox" tabindex="0"></label>`
